Tests/EntitlementsSpec.swift

```diff
@@ -0,0 +1,365 @@
+import Quick
+import Nimble
+import KindeSDK
+import Foundation
+
+class EntitlementsSpec: QuickSpec {
+    override class func spec() {
+        describe("EntitlementsService") {
+            var auth: Auth!
+            var entitlementsService: EntitlementsService!
+            
+            beforeEach {
+                KindeSDKAPI.configure()
+                auth = KindeSDKAPI.auth
+                entitlementsService = auth.entitlements
+            }
+            
+            describe("getEntitlements") {
+                it("returns empty dictionary when no entitlements claim exists") {
+                    // This test assumes the user is not authenticated or has no entitlements
+                    let entitlements = entitlementsService.getEntitlements()
+                    expect(entitlements).to(beEmpty())
+                }
+                
+                it("returns entitlements dictionary when claim exists") {
+                    // This test would require a mock token with entitlements claim
+                    // For now, we test the structure and behavior
+                    let entitlements = entitlementsService.getEntitlements()
+                    expect(entitlements).to(beAKindOf([String: Any].self))
+                }
+            }
+            
+            describe("getEntitlement") {
+                it("returns nil for non-existent entitlement") {
+                    let result = entitlementsService.getEntitlement(featureKey: "non_existent_feature")
+                    expect(result).to(beNil())
+                }
+                
+                it("returns entitlement value when feature exists") {
+                    // This would require a mock token with specific entitlements
+                    // TODO: Add proper test fixtures with mocked tokens
+                    pending("Requires mock token with entitlements claim")
```

Comment:
```suggestion
                    pending("Requires mock token with entitlements claim") {}
```

`pending` requires a closure.